src/olympia/lib/cache.py

```diff
@@ -12,28 +12,41 @@
 from django.conf import settings
 
 
-def make_key(key=None, with_locale=True):
+def make_key(key, with_locale=True, normalize=False):
```

Comment:
If exceeding the key length is a problem maybe we should always normalize the string length (or default to `True`, at least)

Reply:
I'm not a fan of just storing hashes as keys, since they make debugging pretty much impossible. That's why I'd rather use the normalized / hashed version only when necessary.